flume-ng-sources/flume-kafka-source/src/main/java/org/apache/flume/source/kafka/KafkaSource.java

```diff
@@ -453,7 +480,12 @@ private void setConsumerProps(Context ctx) {
       kafkaProps.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
     }
     kafkaProps.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, DEFAULT_AUTO_COMMIT);
-
+    //  The default value of `ssl.endpoint.identification.algorithm` is changed to `https`, since kafka client 2.0+
+    //  And because flume does not accept an empty string as property value,
+    //  so we need to use alternative custom property `ssl.disableTLSHostnameVerification` to check if enable fqdn check.
```

Comment:
This line also greater than 120 characters

Reply:
fixed